client/deposits/overview/instant-deposit-button.js

```diff
@@ -3,24 +3,61 @@
 /**
  * External dependencies
  */
-import { Button, Modal } from '@wordpress/components';
+import { Button, Modal, Notice } from '@wordpress/components';
 import { __, sprintf } from '@wordpress/i18n';
 import { useState } from '@wordpress/element';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
 import Currency from '@woocommerce/currency';
 import apiFetch from '@wordpress/api-fetch';
 
 const currency = new Currency();
 
-const InstantDepositModal = ( { amount, fee, net, onClose, onSubmit, inProgress } ) => {
+const InstantDepositModal = ( {
```

Comment:
This file has 3 React components: the button, the modal, and the error notice. I think it's time to move them to different files inside a separate folder.

Reply:
Done! 😀